CAM Platform/workspaces.md

```diff
@@ -13,812 +13,892 @@ Manage and perform actions on workspaces.
+        "icon": "/services/blobs/download/5bcf2c5c1862a312d7ac3c37/Screen-Shot-2018-10-23-at-16.12.23.png",
+        "id": "workspace5",
+        "name": "WORKSPACE B",
+        "schema": "http://elasticbox.net/schemas/workspaces/team"
+    }
+]
+```
+## POST /services/workspaces
+Creates a workspace and gets the created workspace.
 
-**Request Headers**
+### URL
 
+#### Structure
+```
+[POST] /services/workspaces
+```
+#### Example
+```
+[POST] https://cam.ctl.io/services/workspaces
+```
+### Request
+#### Headers
 ```
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
+#### URI Parameters
+* None
 
-| Parameter | Type |Description |
-|-----------|------|------------|
-| owner | string | User that owns the workspace. |
-| schema | string | Workspace schema. Always “http://elasticbox.net/schemas/workspaces/team” |
-| organizations | array | List of organizations with access to the workspace. |
-| name | string | Workspace name. |
-| icon | string | Workspace icon. |
-| members | array | List of users with access to the workspace. The role is always collaborator. See example request for more details. |
-| ldap_groups | array | List of fully qualified names of LDAP groups that are members of a team workspace. |
+#### Request body parameters
 
-**Request Body**
+| NAME | TYPE | DESCRIPTION | REQ. |
+|------| ---- | ----------- | ---- |
+| costcenter | string | Cost center id | Yes |
+| name | string | Workspace name | Yes |
+| schema | string | Schema Url. For example, http://elasticbox.net/schemas/workspaces/team | Yes |
 
-```
-{
-"schema": "http://elasticbox.net/schemas/workspaces/team",
-"name": "Project Elastic",
-"members": [
-        {
-                "role": "collaborator",
-                "workspace": "david"
-        }
-],
-"owner": "operations"
-}
-```
+### Response
+#### Normal Response Codes
+- **201** Created
+
+#### Common Error Response Codes
+- **400** Invalid Data
+- **401** Unauthorized
+- **409** Conflict
+
+#### Response Parameters
 
 | Parameter | Type |Description |
 |-----------|------|------------|
-| organizations | array | List of organizations with access to the workspace. |
-| updated | string | Date of the last update. |
-| name | string | Workspace name. |
-| created | string | Creation date. |
-| uri | string | Workspace uri. It is: “http://elasticbox.net/schemas/workspaces/team” |
-| members | array | List of users with access to the workspace. |
-| ldap_groups | array | List of fully qualified names of LDAP groups that are members of a team workspace. |
-| owner | string | User that owns the workspace. |
-| icon | string | Workspace icon uri. |
-| id | string | List of users with access to the workspace. |
-| schema | string | Workspace schema uri. |
+| costcenter | string | Cost center id |
+| created | string | Creation date |
+| deleted | boolean | If true, the workspace has been deleted |
+| id | string | Workspace id |
```

Comment:
Please check, I'm not sure if this value will be accepted or directly overriden.